operate/common/src/main/java/io/camunda/operate/connect/ElasticsearchConnector.java

```diff
@@ -145,16 +146,16 @@ public boolean checkHealth(ElasticsearchClient elasticsearchClient) {
     }
   }
 
-  @Bean
-  public RestHighLevelClient esClient() {
+  @Bean("esClient")
+  public ExtendedElasticSearchClient esClient() {
```

Comment:
It looks like the only place that changes the search() call to use the specific ExtendedElasticsearchSearchClient version is in FlowNodeInstanceReader, correct? If so, I would recommend preserving the return type in this config as RestHighLevelClient, preserving the Autowire types elsewhere, and only changing the Autowire type in FlowNodeInstanceReader to ExtendedElasticSearchClient. This will keep us decoupled elsewhere from the specific implementation type.